experimenter/experimenter/templates/changelog/overview.html

```diff
@@ -9,9 +9,13 @@ <h1>Changes Overview</h1>
     <p>Experiment Slug: {{ experiment.slug }}</p>
     {% for changelog in experiment.get_changelogs %}
```

Comment:
So up here, this isn't really a changelog anymore, now it's a group of changelogs and a date?

Reply:
Yes, should I reference it differently ?

Reply:
Yeah I would update the name to reflect what is being looped over.